src/main/java/edu/gatech/cs2340/trydent/animation/Keyframe.java

```diff
@@ -0,0 +1,43 @@
+package edu.gatech.cs2340.trydent.animation;
+
+import edu.gatech.cs2340.trydent.math.BaseVector;
+import edu.gatech.cs2340.trydent.math.Orientation;
+import edu.gatech.cs2340.trydent.math.Position;
+import edu.gatech.cs2340.trydent.math.Scale;
+import edu.gatech.cs2340.trydent.math.curve.Interpolation;
+
+/**
+ * Information holder class for a single keyframe. Here as an inner class
+ * for encapsulation.
+ */
+public class Keyframe extends Orientation {
```

Comment:
Name collision with JavaFX keyframe.
Does it make sense for copy() to return an Orientation?
The methods sounds like clone, but the semantics are a bit different.